projects/swimlane/ngx-ui/src/lib/components/date-range-calendar/date-range-picker.component.html

```diff
@@ -0,0 +1,95 @@
+        <p
+          *ngFor="let preset of presets"
+          [class.active]="selectedPreset === preset.label"
+          (click)="selectPreset(preset)"
+        >
+          {{ preset.label }}
+        </p>
+      </div>
+
+      <!-- Main Calendar + Inputs -->
+      <div class="calendar-panel">
+        <div class="input-row">
+          <div class="input-with-icon">
+            <ngx-input
+              appearance="fill"
+              [(ngModel)]="form.startRaw"
+              (ngModelChange)="onCustomInputChange()"
+              type="text"
+              name="searchInputValue"
+              [placeholder]="placeholders?.start || 'Start'"
+            ></ngx-input>
+
+            <span class="calendar-dialog-btn icon icon-calendar"></span>
+          </div>
+          <span>to</span>
+          <div class="input-with-icon">
+            <ngx-input
+              appearance="fill"
+              [(ngModel)]="form.endRaw"
+              (ngModelChange)="onCustomInputChange()"
+              type="text"
+              [placeholder]="placeholders?.end || 'End'"
+            ></ngx-input>
+            <span class="calendar-dialog-btn icon icon-calendar"></span>
+          </div>
+        </div>
+
+        <div class="validation-error">
+          <span [class.hide]="!validationError">
+            <ngx-icon icon="error_outline"></ngx-icon>
+            {{ validationError || ' ' }}
+          </span>
+        </div>
+
+        <!-- Dual Calendar View -->
+        <div class="dual-calendar-wrapper">
+          <div class="calendar-container">
+            <ngx-calendar
+              name="calendar-range-left"
+              [selectType]="'range'"
+              [range]="rangeModel"
+              [dateLabelFormat]="'YYYY MMM D'"
+              [minDate]="leftMinDate"
+              [maxDate]="leftMaxDate"
+              [(ngModel)]="leftViewDate"
+              (onRangeSelect)="onRangeSelect($event)"
+            ></ngx-calendar>
+          </div>
+
+          <div class="calendar-container">
+            <ngx-calendar
+              name="calendar-range-right"
+              [selectType]="'range'"
+              [range]="rangeModel"
+              [dateLabelFormat]="'YYYY MMM D'"
+              [minDate]="rightMinDate"
+              [maxDate]="rightMaxDate"
+              [(ngModel)]="rightViewDate"
+              (onRangeSelect)="onRangeSelect($event)"
+            ></ngx-calendar>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- Footer Buttons -->
+    <div class="buttons">
+      <div class="left">
+        <ngx-button class="btn btn-bordered" (click)="openSearchStringDocPage()">
+          Read about relative time ranges
+        </ngx-button>
+      </div>
+      <div class="right">
+        <ngx-button class="btn btn-bordered" (click)="onCancel()">Cancel</ngx-button>
+        <ngx-button class="btn btn-primary" [disabled]="!!validationError" (click)="onApply()"> Apply </ngx-button>
+      </div>
+    </div>
```

Comment:
<img width="795" alt="Screenshot 2025-07-08 at 7 34 15 AM" src="https://github.com/user-attachments/assets/8db68876-cc27-48ff-853a-28116c9d63f0" />
Margin is greater than the mockup here.